src/main/resources/application.yml

```diff
@@ -65,6 +65,10 @@ url:
     topic:
       base: "/topic/session"
       pattern: "/topic/session/%s"
+  notification:
+    topic:
+      base: "/user/topic/notification"
+      pattern: "/topic/notification"
```

Comment:
Chyba url będzie inny. Widzę że ludzie subskrybują na `/user/{username}/...` 

Plus nie wiem czy tak trzeba czy to jest konwencja, ale widze ze te user destinations mają `/queue/` zamiast `/topic`. Może warto by to zmienić w takim razie?

https://stackoverflow.com/questions/33720386/how-does-spring-websocket-send-message-to-a-specific-user

Reply:
I jak się zdecydujesz na queue to też z tego co widzę, to trzeba dodać to do `config.enableSimpleBroker`

Reply:
https://www.baeldung.com/spring-websockets-send-message-to-user

Jeszcze takie coś znalazłem, ale szczerze jak to czytam to nie bardzo rozumiem jak te url działają ale są 3 różne zależnie od kontekstu